cypress/e2e/claimApplication/claimantInfo/confirmation.cy.js

```diff
@@ -38,11 +127,144 @@ describe("Confirmation page", () => {
+        I'm requesting your help to complete Form M01 for my
+        New Jersey Temporary Disability Insurance claim (Claim ID: 12114, DOB: Jan 1, 2000).
+        The easiest way to submit it is online at www.nj.gov/labor/MedicalApplicationTDI
+        using this Online Form ID: 25091950002.
+        Fill out the required medical information and submit online.
+        Please submit the form by October 7, 2025 to avoid delays on my claim.
+        Thank you for your help.
+        Liz Lemon
+        lemon@nbc.com
+        (555) 555-5555
+      `;
+
+      const normalizeWhitespace = (text) => text.replace(/\s+/g, ' ').trim();
+      cy.window().then((win) => {
+        return win.navigator.clipboard.readText();
+      }).then((clipboardText) => {
+        expect(normalizeWhitespace(clipboardText)).to.eq(normalizeWhitespace(expectedText));
+      });
+    });
+
+    it("user can copy sample M01 text when session does not have data", () => {
+      cy.clock(new Date(2025, 8, 23)); // 0-indexed; Sept. 23, 2025
+      cy.visit(FIXTURE);
+      cy.get('button.usa-accordion__button[aria-controls="sampleLanguageContainer"]').click();
+      cy.get('#m01CopyButton').click();
+
+      const expectedText = `
+        Subject: Request for Medical Form M01 - Temporary Disability Claim
+        Dear {{ INSERT YOUR MEDICAL PROVIDER'S NAME }},
+        I'm requesting your help to complete Form M01 for my
+        New Jersey Temporary Disability Insurance claim (Claim ID: 12114, DOB: {{ INSERT YOUR DATE OF BIRTH }}).
+        The easiest way to submit it is online at www.nj.gov/labor/MedicalApplicationTDI
+        using this Online Form ID: 25091950002.
+        Fill out the required medical information and submit online.
+        Please submit the form by October 7, 2025 to avoid delays on my claim.
+        Thank you for your help.
+        {{ INSERT YOUR NAME }}
+      `;
+
+      const normalizeWhitespace = (text) => text.replace(/\s+/g, ' ').trim();
+      cy.window().then((win) => {
+        return win.navigator.clipboard.readText();
+      }).then((clipboardText) => {
+        expect(normalizeWhitespace(clipboardText)).to.eq(normalizeWhitespace(expectedText));
+        cy.checkLogEvent(`TDI Confirmation - Copy M01 Sample Clicked`, {});
+      });
+    });
+
+    it("user can download M01 instructions", () => {
+      cy.mockASPX(URL);
+      cy.get('#downloadM01Instructions').click();
+      cy.wait('@aspxSubmission').then(checkM01InstructionsDownload);
+      cy.checkLogEvent(`TDI Confirmation - Download M01 Instructions Clicked`, {});
+    });
+
+    it("shows the no employer message when needed", () => {
+      showElement('DivNoEmps');
+      cy.checkLogEvent(`TDI Confirmation - No Emp Shown`, {});
+      cy.contains('There are no employers that you worked for in the 180 days prior to your first day of disability.')
+        .should('be.visible');
+    });
+
+    it("user can download C01 when award normally shown", () => {
```

Comment:
[dust/pebble]: Thoughts about renaming tests (for this PR at least), so they start with verbs for readability and conciseness? For example: `downloads C01 when award normally shown`